beeline/src/java/org/apache/hive/beeline/BeeLine.java

```diff
@@ -2298,23 +2293,17 @@ public void addLocalDriverClazz(String driverClazz) {
     supportedLocalDriver.add(driverClazz);
   }
 
-  Driver[] scanDrivers(String line) throws IOException {
-    return scanDrivers(false);
-  }
-
-  Driver[] scanDrivers(boolean knownOnly) throws IOException {
```

Comment:
This `knownOnly` flag is never used, so remove it.  By extension, the `scanDrivers(String line)` is also defunct.